lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -243,6 +240,7 @@ def instructor_dashboard_2(request, course_id):
         'certificate_invalidations': certificate_invalidations,
         'generate_certificate_exceptions_url': generate_certificate_exceptions_url,
         'generate_bulk_certificate_exceptions_url': generate_bulk_certificate_exceptions_url,
+        'enable_new_ui_for_csv_reports': new_ui_for_data_download_csv_reports(course_key),
```

Comment:
I think if you want to keep the feature behind a feature flag it should be a generic waffle flag for all courses. Just in case something goes wrong we can easily turn off the feature for all courses.

Reply:
Good point, I can ask this from Seth as well just to be on the safe side.
Thanks for bringing it up.